executor/runtime/docker/docker_env.go

```diff
@@ -31,6 +33,11 @@ export {{ $key }}={{ escapeSQWithFallback $key $val }}
 # export {{ $key }}={{ $val | escape_sq }}
 {{ end -}}
 {{ end -}}
+
+# This path setting makes it convenient for users to run
+# tools that have been injected into the container from the
+# Titus agent.
+export PATH=$PATH:/titus/container-tools/bin
```

Comment:
Do we think our users would be ok with this?  I'm wondering if it's possible to only add this to the path if we're in an ssh session.  Maybe I'm being overly paranoid here. :)

Reply:
It is last in the path, should be ok.